requirements.txt

```diff
@@ -7,4 +7,4 @@ jaxlib==0.1.70
 flax
 bm3d
 # svmbir
-git+https://github.com/cabouman/svmbir.git@nanfix
\ No newline at end of file
+git+https://github.com/cabouman/svmbir.git@master
```

Comment:
I expect the release will be on PyPI quite soon. When it does, this should become `svmbir>=0.2.6`.

Reply:
It's on PyPI now, so this change no longer has to be delayed.

Reply:
Done